camlp4/plan.sh

```diff
@@ -0,0 +1,23 @@
+pkg_name=camlp4
+pkg_origin=core
+pkg_version="4.04"
+pkg_description="Camlp4 is a software system for writing extensible parsers for programming languages. It provides a set of OCaml libraries that are used to define grammars as well as loadable syntax extensions of such grammars."
+pkg_upstream_url="https://github.com/ocaml/camlp4"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('LGPL-2.0')
+pkg_source="https://github.com/ocaml/camlp4/archive/${pkg_version}+1.tar.gz"
+pkg_dirname="${pkg_name}-${pkg_version}-1"
+pkg_shasum="6044f24a44053684d1260f19387e59359f59b0605cdbf7295e1de42783e48ff1"
+pkg_deps=(core/glibc)
```

Comment:
Getting this:
```
[69][default:/src:0]# hab pkg exec core/camlp4 mkcamlp4
/hab/pkgs/core/ocaml/4.04.0/20170510203606/lib/ocaml/extract_crc -I /hab/pkgs/core/camlp4/4.04/20170510204620/lib/camlp4 -I . > /tmp/crc_9c5ea6.ml
ocamlc -I /hab/pkgs/core/camlp4/4.04/20170510204620/lib/camlp4 dynlink.cma camlp4lib.cma /tmp/crc_9c5ea6.ml -I . Camlp4Bin.cmo -linkall
sh: ocamlc: command not found
rm -f /tmp/crc_9c5ea6.ml /tmp/crc_9c5ea6.cmi /tmp/crc_9c5ea6.cmo
sh: rm: command not found
Fatal error: exception Failure("Exit: 127")
```

I think there needs to be a dep on `core/coreutils` and `core/ocaml`.

It works if you make them like this:

```
pkg_deps=(
  core/coreutils
  core/glibc
  core/ocaml
)
pkg_build_deps=(
  core/gcc
  core/make
  core/ocamlbuild
  core/which
)
```